packages/nuxt/src/app/composables/router.ts

```diff
@@ -92,6 +111,23 @@ export const navigateTo = (to: RouteLocationRaw | undefined | null, options?: Na
   }
 
   const toPath = typeof to === 'string' ? to : ((to as RouteLocationPathRaw).path || '/')
+
+  // Early open handler
+  if (options?.open) {
+    if (process.client) {
+      const { target = '_blank', windowFeatures = {} } = options.open
+
+      const features = Object.entries(windowFeatures)
+        .filter(([_, value]) => value !== undefined)
+        .map(([feature, value]) => `${feature.toLowerCase()}=${value}`)
+        .join(', ')
+
+      open(toPath, target, features)
+    }
+
+    return Promise.resolve()
```

Comment:
Currently this will do nothing at all on server but simply return. Maybe worth adding a warning in development mode in this case?

Reply:
I understand the rationale behind it, but it would be ideal if this behavior is automatically ignored on the server side, particularly if it resembles a window.open replication. 

If we use this in the middlewares or setup this will trigger few warnings